testnets/mantrachaintestnet/assetlist.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "mantrachaintestnet",
+  "assets": [
+    {
+      "description": "The native token of MANTRA Chain",
+      "denom_units": [
+        {
+          "denom": "uaum",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "aum",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "uaum",
+      "name": "MANTRA Chain",
+      "display": "AUM",
```

Comment:
'display' value must be a defined denom_unit, but 'AUM' is not defined . Please use 'aum' (lowercase) since that's what's defined.

Reply:
Fixed.